doc/xml/reference.xml

```diff
@@ -1804,7 +1804,7 @@
             <command id="stanza-create" name="Stanza Create">
                 <summary>Create the required stanza data.</summary>
 
-                <text>The <cmd>stanza-create</cmd> command must be run on the host where the repository is located after the stanza has been configured in <file>pgbackrest.conf</file>. If there is more than one repository configured locally on the host, the stanza will be created on each.</text>
+                <text>The <cmd>stanza-create</cmd> command must be run after the stanza has been configured in <file>pgbackrest.conf</file>. If there is more than one repository configured, the stanza will be created on each.</text>
```

Comment:
@pgstef Fair point. @dwsteele I checked all the XML files and there 4 places in reference.xml where will specifically call out `<file>pgbackrest.conf</file>` that could be replaced with something more generic, like `in the pgbackrest configuration file` rather than using the file name - thoughts?

Reply:
Seems pretty clear as is, but I wouldn't object to a separate patch which makes those changes if you both think it would be better. If we decide to do that, then changing it here in this patch is fine since we are already updating the text.

Reply:
Haha, and I didn't see yours! More refresh needed here...

Reply:
"... has been configured in the configuration file" sounds a bit weird. After all, that's our default configuration file. Let's leave it like that, that's fine. Sorry for the noise ;-)